lib/polymer_interop.dart

```diff
@@ -20,98 +20,226 @@ class Polymer {
+  /// `this.children` instead.
+  List<Node> get childNodes => _proxy['childNodes'];
 
+  /// Read-only property that returns a live `List` of the child elements of
+  /// `this`. The list doesn't include text nodes and comments.
   List<Element> get children => _proxy['children'];
 
-  List<Node> get childNodes => _proxy['childNodes'];
-  void set childNodes(List<Node> nodes) {
-    _proxy['childNodes'] = new JsObject.jsify(nodes);
-  }
-
-  Node get parentNode => _proxy['parentNode'];
-  void set parentNode(Node parent) {
-    _proxy['parentNode'] = new JsObject.fromBrowserObject(parent);
-  }
+  // TODO: add docs
+  PolymerClassList get classList => new PolymerClassList(this);
 
+  /// Read-only property that returns the node's first child in the tree, or
+  /// null if the node is childless.
   Node get firstChild => _proxy['firstChild'];
-  void set firstChild(Node child) {
-    _proxy['firstChild'] = new JsObject.fromBrowserObject(child);
-  }
-
-  Node get lastChild => _proxy['lastChild'];
-  void set lastChild(Node child) {
-    _proxy['lastChild'] = new JsObject.fromBrowserObject(child);
-  }
 
+  /// Read-only property that returns the object's first child Element, or null
+  /// if there are no child elements.
   Element get firstElementChild => _proxy['firstElementChild'];
-  void set firstElementChild(Element child) {
-    _proxy['firstElementChild'] = new JsObject.fromBrowserObject(child);
-  }
 
-  Element get lastElementChild => _proxy['lastElementChild'];
-  void set lastElementChild(Element child) {
-    _proxy['lastElementChild'] = child;
-  }
+  /// Sometimes it’s necessary to access the elements which have been distributed
+  /// to a given <content> insertion point or to know to which <content> a given
+  /// node has been distributed. The getDistributedNodes and
+  /// getDestinationInsertionPoints methods, respectively, provide this information.
+  List<Node> getDestinationInsertionPoints() =>
+      _proxy.callMethod('getDestinationInsertionPoints');
 
-  Node get previousSibling => _proxy['previousSibling'];
-  void set previousSibling(Node sibling) {
-    _proxy['previousSibling'] = new JsObject.fromBrowserObject(sibling);
+  /// Sometimes it’s necessary to access the elements which have been distributed
+  /// to a given <content> insertion point or to know to which <content> a given
+  /// node has been distributed. The getDistributedNodes and
+  /// getDestinationInsertionPoints methods, respectively, provide this information.
+  List<Node> getDistributedNodes() => _proxy.callMethod('getDistributedNodes');
+
+  /// Returns the shadow-dom the element is in. Note that the return type is a `DocumentFragment`. The return object will have a property `host`, which is the
+  /// actual custom element.
+  // TODO: the js object does has this 'host` property
+  // the dart object doesn't have this property...
+  DocumentFragment getOwnerRoot() => _proxy.callMethod('getOwnerRoot');
```

Comment:
I see that the Polymer.Base class has a `domhost` property that gives already this exact shortcut:

> **domHost()** Return the element whose local dom within which this element is contained. This is a shorthand for `Polymer.dom(this).getOwnerRoot().host`.

Maybe this shortcut is enough, so that we can leave the PolymerDom wrapper just as this ? I don't know what you're plans for the Polymer.Base class are, but I've made a complete wrapper for this class here: 
https://github.com/kasperpeulen/polymerjs-dart/blob/master/lib/polymerbase.dart

Not sure where a wrapper for the polymbase class would fit, here, or at the polymer-dart repo ? @jakemac53